service/src/templates/communication.community.leads.message.recipient.js

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+var templates = require('./alkemio.template.blocks');
+/* eslint-disable quotes */
+module.exports = () => ({
+  name: 'communication.community.leads.message.recipient',
+  title: '{{messageSender.displayName}} sent a message to your community!',
+  version: 1,
+  channels: {
+    email: {
+      from: '{{emailFrom}}',
+      to: '{{recipient.email}}',
+      replyTo: '{{messageSender.email}}',
```

Comment:
This is exposing the sender's email - is it discussed with Product?

Reply:
![image](https://user-images.githubusercontent.com/23555519/216022966-9bf35c87-1960-43a6-8e08-481ecf2ef141.png)
Yes, this was intentional.